### ISSUE TITLE
linspace/logspace DeprecationWarning cannot be safely interpreted as an integer

### ISSUE THREAD

**Issue by balarsen** [spacepy/spacepy#117]
This is a numpy warning that started recently. 

For anything where an integer is expected numpy is now throwing
`DeprecationWarning: object of type <class 'float'> cannot be safely interpreted as an integer.`

```
import spacepy.toolbox as tb
tb.linspace(1, 5, 10.0)
site-packages/spacepy/toolbox/__init__.py:1310: DeprecationWarning: object of
 type <class 'float'> cannot be safely interpreted as an integer.
  return np.linspace(min, max, num, **kwargs)
Out[3]:
array([1.        , 1.44444444, 1.88888889, 2.33333333, 2.77777778,
       3.22222222, 3.66666667, 4.11111111, 4.55555556, 5.        ])

```

These just need a bit of attention and thought. Options:
1) Cast things within `linspace`/`logspace` into ints 
2) Leave the DeprecationWarning so that the user has control and casts to int

I am good either way. 

Darwin-18.5.0-x86_64-i386-64bit
sys.version_info(major=3, minor=7, micro=3, releaselevel='final', serial=0)
numpy=1.16.2
scipy=1.2.1
matplotlib=3.0.3

**Comment by drsteve** [comment]
We have `num` documented as needing to be integer, so I don't think we need to treat this as an issue.
My preference would be to just leave this so that users get warnings about stuff that might fail in the future (and to alert them to potential issues arising from passing a float in this context, where I believe the default behaviour is currently to truncate).

I recommend closing this as *not an issue*.

**Comment by balarsen** [comment]
Deal, closes #117
